books/microbiology/bake

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '7.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :microbiology) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book, title_element: 'h1')
+  BakeChapterIntroductions.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  book.chapters.each do |chapter|
+    BakePageAbstracts.v1(chapter: chapter)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+  end
+
+  BakeAutotitledNotes.v1(
+    book: book,
+    classes: %w[clinical-focus check-your-understanding micro-connection
+                link-to-learning eye-on-ethics case-in-point disease-profile]
+  )
```

Comment:
Can this be before the `book.chapters` loop on line 25?  If so, you can combine the two `book.chapters` loops (the one on line 25 and the one on line 37.